ports/zeldadoi/zeldadoi/gmloader.json

```diff
@@ -0,0 +1,7 @@
+{
+    "save_dir" : "savedata",
```

Comment:
This folder is missing. Use `.gitkeep` to include an empty folder with git or create it in the `.sh` launch script